### PR TITLE
add Pydantic input classes Context and GuidedPrompt and make few-shot default for all flows

### PR DESCRIPTION
**Pull request by jojortz**

### CODE REVIEW

File: uniflow/flow/model_flow.py
Comment (CambioML) [inline uniflow/flow/model_flow.py:1]: In the `uniflow.model.model` imports, please remove the commented out `OpenAImodel`

File: uniflow/model/model.py [hunk 1]
```diff
@@ -21,21 +23,25 @@ class Model:
...
-        few_shot_template: Dict[str, Any],
+        guided_prompt_template: GuidedPrompt,
         model_config: Dict[str, Any],
     ) -> None:
         """Initialize Model class.
 
         Args:
             model_server (str): Model server name.
-            few_shot_template (Dict[str, Any]): Few shot template.
+            guided_prompt_template (Dict[str, Any]): Guided prompt template.
```
Comment (CambioML) [inline uniflow/model/model.py:33]: Please update the type from `Dict` to `GuidedPrompt`

File: uniflow/schema.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+from typing import Any, Dict, Union
+
+from pydantic import BaseModel, Extra, Field, conlist
+
+logger = logging.getLogger(__name__)
+
+python_to_json_types = {
+    "dict": "object",
+    "list": "array",
+    "tuple": "array",  # JSON doesn't have a direct representation for tuples
```
Comment (CambioML) [inline uniflow/schema.py:13]: Please remove this comment

File: uniflow/schema.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    instruction: str = "Generate one question and its corresponding answer based on the last context in the last example. Following the format of the examples below to include context, question, and answer in the response"
+
+    examples: conlist(Union[Context], min_length=0)
+
+    class Config:
+        extra = Extra.forbid
+
+    def __init__(self, **data):
+        default_prompt_qa_1 = Context(
+            context="""The quick brown fox jumps over the lazy black dog.""",
```
Comment (CambioML) [inline uniflow/schema.py:72]: Please change to double quote

**Review by CambioML — CHANGES_REQUESTED** [review]
Great work overall! I've made some comments for some minor changes, please take a look and let me know if you have any questions

File: uniflow/schema.py (hunk 2 above)
Comment (jojortz) [reply uniflow/schema.py:13]: Done!

**Review by CambioML — APPROVED** [review]
LGTM!

**Pull request merged**